app/components/candidate_interface/continuous_applications/application_review_component.rb

```diff
@@ -53,6 +65,20 @@ def course_info_row
         end
       end
 
+      def qualifications_row
+        {
+          key: 'Qualifications',
+          value: current_course.qualifications_to_s,
+        }
+      end
+
+      def course_length_row
+        {
+          key: 'Course length',
+          value: DisplayCourseLength.call(course_length: current_course.course_length) || 'unset',
```

Comment:
What is the purpose of ` || 'unset'`?

Reply:
I thought that was self explanatory.

Reply:
![Screenshot from 2024-01-29 14-17-12](https://github.com/DFE-Digital/apply-for-teacher-training/assets/135042929/4a2f0f5d-5e2a-4f92-9634-1d8f26e2d309)

Reply:
We could hide the row if Course length is null

Reply:
Then shouldn't it be `DisplayCourseLength.call(course_length: current_course.course_length || 'unset')`

Reply:
I don't think that's a change worth making at this point. I would propose hiding the row if its nil, I'll wait for Content to make a call.

Reply:
Ignore this, I misread `DisplayCourseLength`. Makes no difference where you put the `|| unset`.

Reply:
I think the convention is to render a blank if it's nil. I'll remove the 'unset'